### PR TITLE
Port over odds and ends from sprockets css into webpack

### PR DESCRIPTION
**Pull request by colinxfleming**
I rule and have completed some work on Case Manager that's ready for review!

A small blend of minor css ports, including minor things related to the navbar, flash messages, tooltips, and the multimodal.

Have also deleted a couple of things that we've since moved past; see inline comments.

The only stuff left in there should be forms and button stylings (and other imports); the complexity level goes up there so dealing with those solo.

This pull request makes the following changes:
* port over non-form, non-button, non-import related styling from sprockets app.css

no view changes

It relates to the following issue #s: 
* Bumps #1854 

For reviewer:
* Adjust the title to explain what it does for the notification email to the listserv.
* Tag this PR:
  * `feature` if it contains a feature, fix, or similar. This is anything that contains a user-facing fix in some way, such as frontend changes, alterations to backend behavior, or bug fixes.
  * `dependencies` if it contains library upgrades or similar. This is anything that upgrades any dependency, such as a Gemfile update or npm package upgrade.
* If it contains neither, no need to tag this PR.

### CODE REVIEW

**Review by lomky — APPROVED** [review]
Flagged three things as deleted, but I expect they were meant to be since the app looks ✨ great ✨

File: app/assets/stylesheets/application.css.scss [hunk 1]
```diff
@@ -94,12 +94,6 @@ a:hover, a:active, a:visited, a:focus {
   padding-bottom: 10px;
 }
 
-// empty add - char
-.blush:empty:before {
-  content: '\2013';
-  padding-left: 2px;
-  padding-right: 2px;
-}
```
[the remark this answers is not in the document]
Comment (colinxfleming) [reply app/assets/stylesheets/application.css.scss:102]: yep! this is how we did the `>` on the patient edit page before bootstrap killed its icon system and we moved to fontawesome.

File: app/assets/stylesheets/application.css.scss [hunk 2]
```diff
@@ -145,18 +128,6 @@ a:hover, a:active, a:visited, a:focus {
...
-// Styles for form views
-// using Bootstrap
-// generated by the rails_layout gem
-#error_explanation {
-  @extend .alert;
-  @extend .alert-danger;
-}
-#error_explanation h2 {
-  font-size: 16px;
-}
```
[the remark this answers is not in the document]
Comment (colinxfleming) [reply app/assets/stylesheets/application.css.scss:159]: yep! this is a devise thing generally but we don't really use it.

File: app/assets/stylesheets/application.css.scss [hunk 3]
```diff
@@ -244,18 +215,11 @@ $daria-color-lt : #825fb9;
...
 
 .btn[data-orientation='cancel'] {
   color: red;
 }
 
-#flash {
-  position: fixed;
-}
-
```
[the remark this answers is not in the document]
Comment (colinxfleming) [reply app/assets/stylesheets/application.css.scss:258]: yep! this doesn't do anything in general.

**Pull request merged**